tiledb/dataframe_.py

```diff
@@ -383,7 +384,9 @@ def create_dims(
             dim_filters = tiledb.FilterList(filters)
         elif isinstance(filters, tiledb.FilterList):
             dim_filters = filters
-        elif filters or filters is None:
+        elif filters is None:
+            dim_filters = tiledb.FilterList()
+        elif filters:
             dim_filters = None
         else:
```

Comment:
Is this deliberately inconsistent with the `attr_filters` handling? 
```
if attr_filters is True:
    # default case
    filters = tiledb.FilterList([tiledb.ZstdFilter(-1, ctx=ctx)], ctx=ctx) 
...
elif attr_filters is None:
    filters = None
```

Reply:
Updated